src/containers/Particle.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import MicroParticle from './MicroParticle';
+
+export interface Props {
+  canvas: HTMLCanvasElement;
+  context: CanvasRenderingContext2D;
+  addMicroParticle: Function;
+}
+
+class Particle extends React.Component {
+  canvas: HTMLCanvasElement;
+  context: CanvasRenderingContext2D;
+  random: number = Math.random();
+  random1: number = Math.random();
+  random2: number = Math.random();
+  ID: NodeJS.Timeout;
```

Comment:
I think this should be of type `number` as it's keeping track of a `setTimeout`. I guess the best fix would be to do `window.setTimeout` so TS doesn't freak out.